.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,8 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Added or updated unit **and** E2E tests for my change.
+- [ ] Added or updated unit tests.
+- [ ] Checked with reviewers/maintainers whether E2E tests are needed for my change
```

Comment:
Checking with reviewers/maintainers should not be needed for obvious cases. Only do that when you're unsure.

```suggestion
- [ ] Added or updated e2e tests (check with reviewers/maintainers if you're unsure whether E2E tests are necessary for the change).
```